knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to view Web Session Summary
+description: An article on how to capture and inspect web traffic using Fiddler Everywhere
+type: how-to
+slug: how-to-capture-and-inspect-web-traffic
+tags: capture web traffic, inspect web traffic, web traffic, https, web application
+publish: true
+res_type: kb
+---
+
+## View Sessions Summary
+
+Each __Session__ captured by Fiddler Everywhere has multiple attributes ordered in columns. Use these to identify HTTP related issues, to track and debug your site performance, and to leave custom comments.
+
+## (ID)
+
+An auto-generated identification number for the currently listed sessions.
+
+## Result
+
+Indicates the HTTP status code from the response. A server issues the status codes in response to a client's request made to the server and represented by a three-digit number. The first digit of the status code specifies one of five standard classes of responses.
```

Comment:
A bit confused by the multiple **_server_** mentioning in the second line here, Also I think `and represented` should be `and is represented`

Perhaps something like this would be better?
```
A server issues the status codes in response to a client's request in the form of a three-digit number. 
```